app/msg/[username]/page.js

```diff
@@ -1,15 +1,28 @@
 "use client";
 import React, { useEffect, useState } from "react";
 import { useSession } from "next-auth/react";
-
+import { SEND_MESSAGE } from "@/constants";
 function SendMessage(id) {
   const { data: session } = useSession();
   const [message, setMessage] = useState();
 
   // console.log(id.params.username);
-  useEffect(() => {
-    //check user exists or not
-  }, []);
+
+  async function fetchMessage() {
+    await fetch(`${SEND_MESSAGE}`, {
+      method: "POST",
+      body: JSON.stringify({
+        to: id.params.username,
+        from: { ip: 123, loc: "localhost" },
```

Comment:
This `from` object is hard-coded for testing out api or this will be our implementation?

Same with the `setMessage` at line 22

Reply:
its hardcoded for now will provide dynamic data once fetching of user agent from the senders browser is implemented.